tools/update-readme.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Script to update the README with sponsors details in all packages.
+ *
+ *   node tools/update-readme.js
+ *
+ * @author Milos Djermanovic
+ */
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+import { readFileSync, writeFileSync } from "node:fs";
+import got from "got";
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+const SPONSORS_URL =
+	"https://raw.githubusercontent.com/eslint/eslint.org/main/includes/sponsors.md";
+
+const README_FILE_PATH = "./README.md";
```

Comment:
```suggestion
const README_FILE_PATH = "./README.md";
```
Suggestion: Can we use an absolute path here? If we're testing locally from the "tools" directory and run this file using a relative path, it may throw an error. An absolute path would provide consistency in all cases, ensuring the file is accessed correctly regardless of the current working directory.

But still, we are not going to run it manually so keeping it relative is also okay.

Reply:
since we are not going to run this manually so using a relative path is also okay.